packages/superset-ui-chart-controls/src/components/InfoTooltipWithTrigger.tsx

```diff
@@ -16,19 +16,29 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import React, { CSSProperties } from 'react';
+import React from 'react';
 import { kebabCase } from 'lodash';
-import { Tooltip, OverlayTrigger } from 'react-bootstrap';
 import { t } from '@superset-ui/core';
-
-const tooltipStyle: CSSProperties = { wordWrap: 'break-word' };
+import { Tooltip } from './Tooltip';
 
 export interface InfoTooltipWithTriggerProps {
   label?: string;
   tooltip?: string;
   icon?: string;
   onClick?: () => void;
-  placement?: string;
+  placement?:
+    | 'right'
+    | 'top'
+    | 'left'
+    | 'bottom'
+    | 'topLeft'
+    | 'topRight'
+    | 'bottomLeft'
+    | 'bottomRight'
+    | 'leftTop'
+    | 'leftBottom'
+    | 'rightTop'
+    | 'rightBottom';
```

Comment:
Didn't test AntD's typing, but maybe you can replace this with `TooltipProps['placement']`?

Reply:
Good idea! We can event import `TooltipPlacement` directly